substrate/frame/broker/src/lib.rs

```diff
@@ -172,6 +181,11 @@ pub mod pallet {
 	#[pallet::storage]
 	pub type CoreCountInbox<T> = StorageValue<_, CoreIndex, OptionQuery>;
 
+	/// Keeping track of cores which have auto-renewal enabled.
+	#[pallet::storage]
```

Comment:
You have an invariant that this list is supposed to be sorted. This should the very least be documented here.

Reply:
Yes, keep in mind that this PR is still very much WIP.

Reply:
PR is merged without this being really addressed, should be a 1 liner `fn try_state`